Userland/Libraries/LibVideo/VP9/Parser.h

```diff
@@ -7,8 +7,10 @@
 
 #pragma once
 
+#include <AK/Array.h>
 #include <AK/OwnPtr.h>
 #include <AK/Span.h>
+#include <AK/Vector.h>
```

Comment:
Adding Vector and Array to Parser.h in `LibVideo: Make probability tables save to the specified index` feels out of place

Reply:
Looks like this was because ProbabilitiesTable.h was already including Vector.h, I'll move the inclusion of those to the inter prediction commit where more Vectors and Arrays were added to Parser.h.